notes.js

```diff
@@ -0,0 +1,22 @@
+//for adding styled components
+// const styleLink = document.createElement("link");
+// styleLink.rel = "stylesheet";
+// styleLink.href = "https://cdn.jsdelivr.net/npm/semantic-ui/dist/semantic.min.css";
+// document.head.appendChild(styleLink);
+
+//how to call my proxy from frontend?
+//I need a way to replace calling my service directly
+//i need a url variable
+//depending on...call either proxy or service
+//env variables
+//default behavior = my service campId = 0
+//special cases send to proxy
+//use react router
+//grab url off product itself
+
+//read about env variables webpack
+//NODE_ENV special cuz express and happy say its important
+//use lighthouse to measure my performance
+  //differences in production vs. non-production????
+
+  // RUN_PROXY_OR_NOT
```

Comment:
This is smart, I've been leaving all my notes IN my code and it was getting real messy! Great idea!